synthnn/performance/cpu_backend.py

```
@@ -21,6 +21,11 @@ class CPUBackend(ComputeBackend):
     def __init__(self, device_id: int = 0):
         super().__init__(device_id)
         self.num_threads = multiprocessing.cpu_count()
+
+    @property
+    def backend_type(self):
```

Ruff (0.11.9)

Missing return type annotation for public function `backend_type`

(ANN201)